.github/workflows/flake8.yml

```
@@ -27,7 +27,7 @@ jobs:
         with:
           persist-credentials: false
       - name: Flake8 Scan
-        uses: reactive-firewall/flake8-cq@82fb2d7f000551444b102a9bd715b06ad4ce8a01  # v1
+        uses: reactive-firewall/flake8-cq@33e74e459ac2d585295585c556882f20dcdcbac5  # v1
```

`33e74e459ac2d585295585c556882f20dcdcbac5` is version v1.5

```suggestion
        uses: reactive-firewall/flake8-cq@33e74e459ac2d585295585c556882f20dcdcbac5  # v1.5
```